docs/guides/mobile-plugin-gma-android.md

```diff
@@ -31,7 +31,7 @@ To run this plugin, install the following:
 
 - [Google Mobile Ads SDK v22.0.0+](https://developers.google.com/admob/android/sdk)
 - [UID2 SDK for Android v0.4.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk) (for documentation, see [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
```

Comment:
For Android versions, can we please bump to v0.5.0

Reply:
Done see update thx! Also see additional files added to the PR.